<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>passback</class>
 <widget class="QDialog" name="passback">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>110</y>
     <width>501</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>101</y>
      <width>261</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>50</y>
      <width>261</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_newpass">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>152</y>
      <width>261</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cnewpass">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>203</y>
      <width>261</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>50</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Name </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>101</y>
      <width>38</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>152</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>203</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	position: absolute;
width: 195px;
height: 66px;
left: 368px;
top: 716px;

	background-color: qlineargradient(spread:pad, x1:0, y1:0.625, x2:1, y2:1, stop:0 rgba(135, 164, 255, 255), stop:1 rgba(255, 140, 140, 255));
border-radius: 15px;

}
*:hover{
color: #f5E0E0;
}</string>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eye">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>155</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ceye">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>205</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>611</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
left: 1.97%;
right: -24.05%;
top: 12.78%;
bottom: 77.5%;

font-family: Jua;
font-style: italic;
font-weight: 400;
font-size: 45px;
line-height: 76px;
text-align: center;

color: #FFFFFF;
</string>
   </property>
   <property name="text">
    <string>Routine Management System</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>587</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
left: 0%;
right: -37.9%;
top: 0%;
bottom: 9.62%;

background: #4E61C8;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>0</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 551px;
height: 69px;
left: 198px;
top: 239px;

font-family: 'Jua';
font-style: normal;
font-weight: 400;
font-size: 48px;
line-height: 60px;
text-align: center;

color: #000000;
</string>
   </property>
   <property name="text">
    <string>Forgot Password?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>110</y>
     <width>651</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-image: url(:/img/IMG resources/Rectangle 10.png);
position: absolute;
width: 660px;
height: 70px;
left: 144px;
top: 559px;
background-repeat:no-repeat;

border-radius: 50px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>611</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
left: 1.97%;
right: -24.05%;
top: 12.78%;
bottom: 77.5%;

font-family: Jua;
font-style: italic;
font-weight: 400;
font-size: 45px;
line-height: 76px;
text-align: center;

color: #FFFFFF;
</string>
   </property>
   <property name="text">
    <string>Routine Management System</string>
   </property>
  </widget>
  <zorder>label_10</zorder>
  <zorder>label_4</zorder>
  <zorder>label_2</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
